scripts/gh-actions/parse_versions.py

```diff
@@ -12,7 +12,7 @@ def parse_versions(input_versions):
     return json_array
 
 if __name__ == "__main__":
-    if len(sys.argv) < 1:
+    if len(sys.argv) < 2:
```

Comment:
Does this work as expected? I ran 
```
╰─$  GITHUB_OUTPUT="output"  python ./scripts/gh-actions/parse_versions.py "" 
bundle_versions=[""]
```
Shouldn't it fail in this case?

Reply:
I have added a clause to the if statement that checks for an empty string, not sure if there is a smarter way to do this